XRTK-Core/Packages/com.xrtk.core/Runtime/Interfaces/InputSystem/IMixedRealityPointer.cs

```diff
@@ -48,11 +47,6 @@ public interface IMixedRealityPointer : IEqualityComparer
         /// </summary>
         ICursorModifier CursorModifier { get; set; }
 
-        /// <summary>
-        /// The currently active teleport hotspot.
-        /// </summary>
-        IMixedRealityTeleportHotSpot TeleportHotSpot { get; set; }
```

Comment:
Not replaced with ITeleportAnchor?

Reply:
Huh? Let me check this I updated ALL of the references and specifically searched for the string "Hotspot" and updated.

Reply:
Ah, I misread this. This is fine. Pointers do no longer have a reference to hotspots/anchors in their interface. So this change is correct.

Reply:
Interesting, how do we know which anchor is active for the specific pointer then?

Reply:
The active anchor is available via `ITeleportTargetProvider.cs`, which happens to be implemented by the `TeleportPointer`. Generally providing a teleport target is not tied to pointers. Anything implementing the mentioned interface can provide teleport targets.

Reply:
This was a convenience property for those consuming pointer event data, so they knew which specfic anchor the pointer was targeting

Reply:
> The active anchor is available via ITeleportTargetProvider.cs

Sounds an awful lot like that pointer mediator 😂

I'll late a look and get back to you.

Reply:
It does not. It's simply an interface for components providing a teleport target. You could be throwing a ball and where it lands that would be your teleport target. See how that is not a pointer?

Reply:
You need to let go of the idea that only pointers provide teleportation targets.

Reply:
@FejZa That's a fair point 😅